src/common/store/accounts.slice.test.ts

```diff
@@ -0,0 +1,75 @@
+import { fAccount, fPrivateKey } from '@fixtures';
+import type { SerializedWallet, TAddress } from '@types';
+import { WalletType } from '@types';
+
+import slice, {
+  addAccount,
+  fetchAccounts,
+  fetchFailed,
+  fetchReset,
+  removeAccount,
+  setGeneratedAccount,
+  updateAccount
+} from './accounts.slice';
+
+const wallet: SerializedWallet = {
+  walletType: WalletType.PRIVATE_KEY,
+  privateKey: fPrivateKey
+};
+
+describe('AccountSlice', () => {
+  it('addAccount(): adds account to state', () => {
```

Comment:
This is a bit easier to read if you move the different functions into a new describe block (e.g., `describe('addAccount', () => { it(...); }`)